src/presign/participant.rs

```diff
@@ -977,12 +975,18 @@ impl PresignKeyShareAndInfo {
         sender_r1_priv: &RoundOnePrivate,
         receiver_r1_pub_broadcast: &RoundOnePublicBroadcast,
     ) -> Result<(RoundTwoPrivate, RoundTwoPublic)> {
-        // Picking betas as elements of [+- 2^384] here is like sampling them from the
-        // distribution [1, 2^256], which is akin to 2^{ell + epsilon} where ell
-        // = epsilon = 384. Note that we need q/2^epsilon to be negligible.
-        let beta = random_plusminus_by_size(rng, ELL);
-        let beta_hat = random_plusminus_by_size(rng, ELL);
-
+        let beta = random_plusminus_by_size(rng, ELL_PRIME);
```

Comment:
Suggestion: I'd suggested changing the name of the method to `sample_random_from_plusminus_by_size`, or something that highlight what `random` refers to.

Reply:
I'm going to punt this to #145 because this will touch a bunch of code unrelated to this PR. I would love to be able to call this with `sample::random_plusminus_by_size()` so there's a verb involved.